schema.graphql

```diff
@@ -82,8 +108,8 @@ type Coin @jsonField {
 
 type NativeTransfer @entity {
   id: ID!
-  toAddress: String! @index
-  fromAddress: String! @index
+  to: Account! @index
+  from: Account! @index
```

Comment:
I think we should use "sender" and "recipient" to be consistent with the bank module's nomeclature.